Homework1/MatrixMultiplication/MatrixMultiplication/Matrix.cs

```diff
@@ -0,0 +1,202 @@
+﻿/// <summary>
+/// Class for matrices and actions on matrices
+/// </summary>
+public class Matrix
+{
+    private int[,] matrix;
+
+    /// <summary>
+    /// Amount of rows in matrix
+    /// </summary>
+    public int Rows 
+    {
+        get { return this.matrix.GetLength(0); } 
+    }
```

Comment:
```suggestion
    public int Rows => this.matrix.GetLength(0);
```
Совсем .NET забыли за лето :)